### PR TITLE
FE-166 How it works section

### PR DESCRIPTION
**Pull request by jbianciotto**
# Description
 How it works sections on Home page

# Jira tickets
https://trialtech.atlassian.net/browse/FE-166

# Screenshots

If you think a screenshot would be helpful to show this change, please attach it here :grin:

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue) :bug:
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected) :bomb::boom:
- [ ] Configuration / Setup :wrench:
- [ ] Clean up / Technical debt :hammer:

## Apps affected by this change

- [x] Web
- [ ] Landing
- [ ] Management portal

## Project code modified by this change

- [X] Web
- [ ] Landing
- [ ] Management portal
- [ ] Packages

# Checklist:

- [X] I have performed a self-review of my own code :innocent:
- [X] My code is clean and it follows the guidelines of this project :nail_care:
- [X] My changes don't generate new warnings :ok_hand:

### CODE REVIEW

File: apps/web/app/[lang]/howitworks/HowItWorks.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  locale,
+}) => {
+  return (
+    <div className="flex flex-col sm:flex-row gap-4">
+      <H2 label={'¿Como funciona?'} className="!mt-0" />
+      {steps.map((step, index) => {
+        return (
+          <Fragment key={index}>
+            {index != 0 && (
+              <div className="w-full h-0 sm:w-px sm:h-auto sm:mx-4 border border-gray-dark border-opacity-30"></div>
```
Comment (libattistella) [inline apps/web/app/[lang]/howitworks/HowItWorks.tsx:37]: Is there any reason to have an empty `div`? Maybe we can use paddings or margins in the steps

File: apps/web/app/[lang]/howitworks/HowItWorks.types.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+export type StepDefinition = {
+  title: string;
+  description: string;
+};
```
Comment (libattistella) [inline apps/web/app/[lang]/howitworks/HowItWorks.types.ts:4]:
```suggestion
};

```

File: apps/web/app/[lang]/howitworks/HowItWorks.types.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+export type Step = {
+  icon: string;
+  es?: StepDefinition;
```
Comment (libattistella) [inline apps/web/app/[lang]/howitworks/HowItWorks.types.ts:7]: Remove this es/en/pt once the JSON file is refactored

File: apps/web/app/[lang]/page.tsx [hunk 4]
```diff
@@ -2,15 +2,30 @@ import { Layout } from '@components/core/layout/Layout';
 import {
   defaultLocale,
   unstable_setRequestLocale,
-  useTranslations,
+  getTranslations,
```
Comment (libattistella) [inline apps/web/app/[lang]/page.tsx:5]: Which is the difference between them?

**Review by libattistella — CHANGES_REQUESTED** [review]
Nice work, a few things

File: apps/web/app/[lang]/howitworks/HowItWorks.tsx (hunk 1 above)
Comment (jbianciotto) [reply apps/web/app/[lang]/howitworks/HowItWorks.tsx:37]: The empty div is the separator, it is just a grey vertical line

File: apps/web/app/[lang]/page.tsx (hunk 4 above)
Comment (libattistella) [reply apps/web/app/[lang]/page.tsx:5]: I see, good to know, thanks!

File: apps/web/intl/locales/en.json [hunk 5]
```diff
@@ -39,7 +39,10 @@
     }
   },
   "home": {
-    "lang": "English"
+    "lang": "English",
```
Comment (libattistella) [inline apps/web/intl/locales/en.json:42]: You can already remove this line as part of this PR. The current content of the home page won't be there aymore

**Review by libattistella — CHANGES_REQUESTED** [review]

File: apps/web/app/[lang]/page.tsx [hunk 6]
```diff
@@ -2,15 +2,30 @@ import { Layout } from '@components/core/layout/Layout';
...
 
-export default function Page({ params: { lang = defaultLocale } }) {
+export default async function Page({ params: { lang = defaultLocale } }) {
   unstable_setRequestLocale(lang);
-  const t = useTranslations('home');
+  const t = await getTranslations('home');
+
+  //TODO: replace with real fetch
+  const getHowItWorksSteps = async () => {
+    const team = await fs.readFile(
```
Comment (libattistella) [inline apps/web/app/[lang]/page.tsx:17]:
```suggestion
    const steps= await fs.readFile(
```

File: apps/web/app/[lang]/page.tsx [hunk 7]
```diff
@@ -2,15 +2,30 @@ import { Layout } from '@components/core/layout/Layout';
...
+      process.cwd() + '/api/mocks/howItWorksSteps.json',
+      'utf8',
+    );
+    return JSON.parse(team);
```
Comment (libattistella) [inline apps/web/app/[lang]/page.tsx:21]:
```suggestion
    return JSON.parse(steps);
```

File: apps/web/app/[lang]/howitworks/Step.tsx [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+export const Step: React.FC<StepProps> = ({
+  icon,
+  title,
+  description,
+  addSeparator = true,
+}) => {
+  return (
+    <div
+      className={clsx(
+        'block sm:flex sm:flex-col sm:gap-6 sm:px-4 ',
```
Comment (libattistella) [inline apps/web/app/[lang]/howitworks/Step.tsx:14]:
```suggestion
        'block sm:flex sm:flex-col sm:gap-6 sm:px-4',
```

File: apps/web/app/[lang]/howitworks/HowItWorks.tsx [hunk 9]
```diff
@@ -0,0 +1,27 @@
+import { useTranslations } from 'intl';
+import { H2 } from 'ui/core';
+import { Step } from './Step';
+
+import { HowItWorksProps } from './HowItWorks.types';
+import { getIcon } from './helpers';
+
+export const HowItWorks: React.FC<HowItWorksProps> = ({ steps }) => {
+  const t = useTranslations('home.howitworks');
```
Comment (libattistella) [inline apps/web/app/[lang]/howitworks/HowItWorks.tsx:9]:
```suggestion
  const t = useTranslations('home.howitworks');
  
  if (!steps || steps.length === 0) return null;
```

**Review by libattistella — CHANGES_REQUESTED** [review]

**Pull request merged**